website/src/components/props-table.tsx

```diff
@@ -41,24 +42,48 @@ const PropsTable = ({
   const variantValues =
     themeComponent?.variants && Object.keys(themeComponent.variants)
 
+  const extendThemeLink = (
+    <Link href="/customize-theme#customizing-component-styles" passHref>
```

Comment:
This link leads to a 404 page, please see https://chakra-ui-l4wyzwok3.vercel.app/docs/components/link